app/models/quiz.rb

```diff
@@ -94,4 +100,70 @@ def is_katakana?(text)
         return true if text =~ /\A[ァ-ヶー－]+\z/
         false
     end 
+
+    def pokemon_species
+        url = "#{BASE_URL}#{POKEMON_SPECIES_URL}#{self.pokemon_id.to_s}"
+        client = HTTPClient.new                 # インスタンスを生成
+        response = client.get(url)   
+        results=JSON.parse(response.body) 
```

Comment:
[nits]
`results` の変数は利用していないので無くても良いと思います！

```suggestion
        JSON.parse(response.body) 
```